packages/clerk-js/src/ui/components/SignIn/SignIn.tsx

```diff
@@ -132,7 +132,10 @@ function SignInRoutes(): JSX.Element {
               </Route>
               {signInContext.withSessionTasks && (
                 <Route path={SESSION_TASK_ROUTE_BY_KEY['org']}>
-                  <SessionTask task='org' />
+                  <SessionTask
+                    task='org'
+                    redirectUrlComplete={signInContext.afterSignUpUrl}
```

Comment:
❓  Not `afterSignInUrl`

Reply:
This is within the `isCombinedFlow` routes, so those are the sign-up routes rendered within `SignIn` component therefore it uses `signUpContext`